### ISSUE TITLE
Add prop to get wrapped component ref

### ISSUE THREAD

**Issue by ksummerlin** [kentor/react-click-outside#20]
It would be a nice addition to have a property like wrappedRef to be able to retrieve the WrappedComponent from the calling code.

Something similar to the [HOC example](https://facebook.github.io/react/docs/higher-order-components.html#refs-arent-passed-through).

**Comment by kentor** [comment]
Is there a specific use case in mind?

**Comment by hatched** [comment]
For some time now we have been running a minimally modified local copy of this module to expose the `WrappedComponent` for easier testing. 

```js
describe('Dropdown Menu', function() {
  it('can render', () => {
    const renderer = jsTestUtils.shallowRender(
      <DropdownMenu.wrappedComponent
        handleClickOutside={sinon.stub()}>
        <li>child</li>
      </DropdownMenu.wrappedComponent>, true);
    const output = renderer.getRenderOutput();
    const expected = (
      <Panel instanceName="dropdown-menu" visible={true}>
        <ul className="dropdown-menu__list">
          <li>child</li>
        </ul>
      </Panel>
    );
    expect(output).toEqualJSX(expected);
  });
});
```

**Comment by hatched** [comment]
I've created PR #25 to commit this change upstream.

**Comment by kentor** [comment]
Thanks. I have a use case for this now so I will implement something like `wrappedRef` property.

**Comment by kentor** [comment]
released in v3.0.0
